src/Pulsar.Client/Internal/HttpLookupService.fs

```diff
@@ -0,0 +1,217 @@
+﻿namespace Pulsar.Client.Internal
+
+open System.IO
+open System.Net.Http
+open System.Text.Json.Serialization
+open Pulsar.Client.Api
+open Pulsar.Client.Common
+open System
+open System.Net
+open FSharp.UMX
+open System.Text.Json
+open Microsoft.Extensions.Logging
+open Pulsar.Client.Schema
+
+type internal HttpLookupService (config: PulsarClientConfiguration, _connectionPool: ConnectionPool) =
+
+    let httpClient = new HttpClient(new SocketsHttpHandler(
+        PooledConnectionLifetime = TimeSpan.FromSeconds(30),
```

Comment:
I suggest we keep it at 2 minutes as suggested in docs
https://learn.microsoft.com/en-us/dotnet/fundamentals/networking/http/httpclient-guidelines#recommended-use

Reply:
Fixed